blocks/feed/feed.js

```diff
@@ -141,6 +143,14 @@ export default async function decorate(block) {
   block.innerHTML = '';
   const blockContents = resultParsers[blockType](results, blockCfg);
   const builtBlock = buildBlock(blockType, blockContents);
+  if (feedTags && blockContents.length) {
```

Comment:
I thing rather than being a part of the `feed` block, this is specific to the `/tags` page and we don't need it elsewhere. So I'd suggest adding a page level script which only runs for `tags` page and does this.

Reply:
Do you mean we will specify something in the page level metadata and on the basis of that we will add our logic in scripts.js ?

Reply:
I know this logic is a little bit tricky, the reason why I put it here is that, I don't want to make multiple times of the query. It's very heavy for rendering the page. Any suggestions?